app/build.gradle.kts

```diff
@@ -55,6 +55,8 @@ kapt {
 
 dependencies {
 
+    api(project(":base"))
```

Comment:
Could you create `Modules` object under `buildSrc` and put this on it? This can be better If we call like that; `api(project(Modules.base))`